CHANGELOG

```diff
@@ -1,6 +1,9 @@
 v3.6.1 (2020-XX-XX)
 -------------------
 
+* Fixed an edge case in VelocyPack when the padding of a 1-byte offsetSize array 
+  is removed but the first few entries of the array contain a Slice of type None.
```

Comment:
Should we add that this is not reachable?

Reply:
I would leave it as it is, as in theory something else could produce VelocyPack None values.
I don't know what exactly could trigger this, but in theory it could have happened. At least it happened to us when setting the profiling level to that undocumented value. :smile: